packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -210,14 +210,41 @@ async function getNpmTarballUrl(packageName: string, version: string = 'latest')
   return stdout.trim();
 }
 
+/**
+ * Gets expo SDK version major from the local package.json.
+ */
+async function getLocalSdkMajorVersion(): Promise<string | null> {
+  const path = require.resolve('expo/package.json', { paths: [process.cwd()] });
+  if (!path) return null;
+  const { version } = require(path) ?? {};
```

Comment:
What do you think if we wrap it in try/catch and handle it nicely? Maybe an appropriate console.error, but with a fallback to `latest` anyway?

Reply:
I've added a try around calling this function